app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -158,7 +163,8 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
                             confirmColor = R.color.primary
                         ) {
                             binding.txConfirmButton.isEnabled = false
-                            viewModel.submitConfirmation(viewModel.txDetails!!)
+//                            viewModel.submitConfirmation(viewModel.txDetails!!)
+                            viewModel.submitRejection()
```

Comment:
Are you just using the the UI for confirmation to test?

Reply:
yes, ui part is in the separate pr. We should merge it first

Reply:
This has now UI.